### DISCUSSION TITLE
Hidden panels in the overview dashboard?

### DISCUSSION THREAD

**Discussion by mljdivemaster** [ZachChristensen28/pihole_dns_app#28] [Ideas]
I have two PiHole Servers and the overview dashboard shows panels that is not populated:
![image](https://user-images.githubusercontent.com/19711259/105011506-532e9a00-5a3d-11eb-8836-8161afd1b035.png)

Until I select a PiHole Server, can't we have a panel that shows this:
![image](https://user-images.githubusercontent.com/19711259/105011389-309c8100-5a3d-11eb-9850-d4cace470f93.png)

It can be controlled with tokens that changes based on what you select in the dropdown menu. 

Do you want me to make an example in a branch?

**Comment by ZachChristensen28** [comment]
Yes, please do.

**Reply by mgjh** [reply]
Will do
